src/sources/logplex.rs

```diff
@@ -65,6 +65,7 @@ impl SourceConfig for LogplexConfig {
 
         let routes = svc.or(ping);
 
+        info!(message = "building logplex server", addr = ?self.address);
```

Comment:
```suggestion
        info!(message = "building logplex server", addr = %self.address);
```
I think we still want `Display` here?